data/converted/folder_png.cpp

```diff
@@ -0,0 +1,102 @@
+// this file was auto-generated from "folder.png" by res2h
```

Comment:
Idea: maybe make this folder image greyscale, so that themers can colorize them to their wishes via the `<color>` tag?

Reply:
Your wish is my command

Reply:
I wasn't sure how to properly allow themes to change icon color so I added a tag to completely change the icon, see my last comment for more informations.